### PR TITLE
Patch meson.build to use install_rpath

### LINKED ISSUE #3: ldconfig produces erroneous links for liborc and liborc-test
`ldconfig` does something funky with the libraries, creating links named `'\'` and `$'\003'`.
Interestingly enough, I couldn't reproduce that on my host OS but could -- albeit with the same `glibc` version -- in a `fedora:31` container:
```sh
~ >>>  podman run --rm -it centos:6 sh -c 'cd /tmp && curl -sL https://anaconda.org/conda-forge/gstreamer-orc/0.4.31/download/linux-64/gstreamer-orc-0.4.31-h516909a_0.tar.bz2 | tar -xjf- lib/liborc-test-0.4.so.0.31.0 && ldconfig --version | head -n1 && ldconfig -vnN ./lib'
ldconfig (GNU libc) 2.12
./lib:
	\ -> liborc-test-0.4.so.0.31.0 (changed)
~ >>>  podman run --rm -it fedora:31 sh -c 'dnf -qy install bsdtar && cd /tmp && curl -sL https://anaconda.org/conda-forge/gstreamer-orc/0.4.31/download/linux-64/gstreamer-orc-0.4.31-h516909a_0.tar.bz2 | bsdtar -xf- lib/liborc-test-0.4.so.0.31.0 && ldconfig --version | head -n1 && ldconfig -vnN ./lib'
ldconfig (GNU libc) 2.30
./lib:
	\ -> liborc-test-0.4.so.0.31.0 (changed)
~ >>>  podman run --rm -it archlinux/base sh -c 'cd /tmp && curl -sL https://anaconda.org/conda-forge/gstreamer-orc/0.4.31/download/linux-64/gstreamer-orc-0.4.31-h516909a_0.tar.bz2 | bsdtar -xf- lib/liborc-test-0.4.so.0.31.0 && ldconfig --version | head -n1 && ldconfig -vnN ./lib'              
ldconfig (GNU libc) 2.30
./lib:
	liborc-test-0.4.so.0 -> liborc-test-0.4.so.0.31.0 (changed)
```
I have no idea what the cause of this is.

xref: https://github.com/conda-forge/libvips-feedstock/pull/4#issuecomment-559031676

### PR DESCRIPTION
**Pull request by mbargull**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

fixes gh-3

Thanks to @kleisauke for the patch.

cc @sebastian-luna-valero, @kleisauke

### CODE REVIEW

**Comment by mbargull** [comment]
Before merging: Can we please fix the `about/license` here?
Skimming over `COPYING`, this looks like `BSD-2-Clause AND BSD-3-Clause` (<-- SPDX expression), right?

**Comment by mbargull** [comment]
And does it make sense to add a (regression) test for this? E.g.,
```yaml
test:
  requires:
    - chrpath  # [linux]
   commands:
    - chrpath -l ${PREFIX}/lib/liborc-0.4${SHLIB_EXT} | grep -qF 'RPATH=$ORIGIN/.'  # [linux]
    - chrpath -l ${PREFIX}/lib/liborc-0.4-test${SHLIB_EXT} | grep -qF 'RPATH=$ORIGIN/.'  # [linux]
```

File: recipe/meta.yaml
Comment (mbargull) [inline recipe/meta.yaml:36]: Please double-check this!

Comment (sebastian-luna-valero) [reply recipe/meta.yaml:36]: Here is the licensing:
https://gitlab.freedesktop.org/gstreamer/orc/blob/master/COPYING

I think it should be custom, shouldn't it?

Comment (kleisauke) [reply recipe/meta.yaml:36]: The Cerbero build system (which is the official build system for GStreamer) identifies the orc license as BSD-like. See: https://gitlab.freedesktop.org/gstreamer/cerbero/blob/69708f677d1d558ce80d5e31d5622132795aa212/recipes/orc.recipe#L17

We could do something similar:
```suggestion
  license: BSD-like
  license_family: BSD
```

Comment (sebastian-luna-valero) [reply recipe/meta.yaml:36]: Thanks @kleisauke 

Would you like to be included as a maintainer of this recipe?

Comment (kleisauke) [reply recipe/meta.yaml:36]: @sebastian-luna-valero Thanks for asking. I don't use Conda on a daily basis, so I'm probably not good at maintaining this recipe.

As an aside, thanks for maintaining the libvips recipe!

Comment (mbargull) [reply recipe/meta.yaml:36]: My point was that the license does not seem to be "custom", really, but just looks to be `BSD-2-Clause AND BSD-3-Clause`. That `LicenseA AND LicenseB` is a SPDX license expression (see https://spdx.org/ids-how for details) and we (as in parts of conda-forge and others in the Conda ecosystem) are slowly pushing to use those expressions for quite some time.

The only thing that is ambiguous about the license are the very first words: "The majority of ...". That wording is unfortunate because it is **not** clear if they mean "Apart from the exceptions mentioned below ..." with this or if the license is in fact `BSD-2-Clause AND BSD-3-Clause AND some-other-unnamed-licenses`.

Comment (kleisauke) [reply recipe/meta.yaml:36]: Ignore my comment about the BSD-like change (it was due to the first words of the license). The `BSD-2-Clause AND BSD-3-Clause` SPDX license expression is more correct and less vague.

(I think we can (safely) interpret the first words as _"With the exception of source that contains the Mersenne Twister algorithm ..."_)

**Review by sebastian-luna-valero — APPROVED** [review]
Many thanks both!

**Pull request merged**
